dspace-api/src/main/java/org/dspace/eperson/Group.java

```diff
@@ -203,7 +202,9 @@ public String getName()
 
     public void setName(Context context, String name) throws SQLException
     {
-        getGroupService().setMetadataSingleValue(context, this, MetadataSchema.DC_SCHEMA, "title", null, null, name);
+        if (!permanent)
+            getGroupService().setMetadataSingleValue(context, this,
+                    MetadataSchema.DC_SCHEMA, "title", null, null, name);
```

Comment:
I would move this "business" logic to the service layer, it doesn't belong in a database table representation class.